entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
> The EF6 warning shouldn't be the first content, it should come after the intro. That's what I did in my rewrite. It should be an [INCLUDE] so it's only written once, not copy/paste everywhere.

We are actually thinking about not having that note on every page any more. Instead we want to prefix titles in most EF Core documentation pages with "EF Core |" which will help with search: 

https://github.com/aspnet/EntityFramework.Docs/issues/362